templates/contact-us.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <section class="p-strip--image is-dark is-deep u-no-padding--top u-no-padding--bottom u-extra-space">
+    <div class="p-strip--suru-background">
+      <div class="row">
+        <div class="col-6">
+          <img src="https://assets.ubuntu.com/v1/6685f10c-contact-canonical-icon.svg" alt="Canonical contact icon">
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="p-strip u-extra-space">
+    <div class="row">
+      <div class="col-5">
+        <p class="p-heading--three">Sales</p>
+        <ul class="p-list--divided u-sv1">
+          <li class="p-list__item">Americas<a class="u-float-right" href="tel:+17372040291">+1 737 2040291</a></li>
+          <li class="p-list__item">France<a class="u-float-right" href="tel:+33184889319">+33 184889319</a></li>
+          <li class="p-list__item">Germany<a class="u-float-right" href="tel:+4961512746816">+49 615 127 46816</a></li>
+          <li class="p-list__item">Japan<a class="u-float-right" href="tel:+81362053075">+81 3 6205 3075</a></li>
+          <li class="p-list__item">Spain<a class="u-float-right" href="tel:+34932201131">+34 932 201131</a></li>
+          <li class="p-list__item">UK<a class="u-float-right" href="tel:+44203 6565291">+44 203 656 5291</a></li>
+        </ul>
+        <p class="p-heading--three">Online</p>
+        <p class="u-sv3"><a href="">Get in touch</a> about open source infrastructure, applications, development and operations. We are proud to deliver the world’s best pathway to open source success.</p>
```

Comment:
there is no form? perhaps use this until we make one? - https://ubuntu.com/contact-us/form?product=generic-contact-us